tests/cosmosLightClient/test-verifier-in-cosmos-relay.ts

```diff
@@ -65,10 +64,13 @@ describe('Light Client Verifier In Cosmos', () => {
     );
 
     const defaultInitHeaderRoot =
-      '0xc43d94aaea1342f8e551d9a5e6fe95b7ebb013142acf1e2628ad381e5c713316';
+      '0x4ce76b7478cb0eee4a32c7f25bb561ca1d0f444d1716c8f6f260900ef45f37d2';
```

Comment:
Use the first update file we have to read this header root, in order to avoid hard coded values like this

Reply:
fixed in was commit